From 82f9446500df6dfece773596d5c6aef05f98a998 Mon Sep 17 00:00:00 2001
From: Mikael Persson <mikael.persson@intel.com>
Date: Fri, 19 Dec 2014 23:22:21 +0100
Subject: [PATCH 3/3] atomisp2: build also css2401a0 version.

If CONFIG_VIDEO_ATOMISP is set, build atomisp
driver both for css2400b0 (BYT) and css2401a0 (CHT).

Change-Id: I8474df3409a0584449e195b5f2ff27482c2eac89
Signed-off-by: Mikael Persson <mikael.persson@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/Makefile     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile
index 517d597..77ce21b 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile
@@ -1,6 +1,6 @@
 ifdef CONFIG_GMIN_INTEL_MID
 obj-$(CONFIG_VIDEO_ATOMISP) += css2400b0_v21_build/
-#obj-$(CONFIG_VIDEO_ATOMISP) += css2401a0_v21_build/
+obj-$(CONFIG_VIDEO_ATOMISP) += css2401a0_v21_build/
 else
 ifeq ($(CONFIG_64BIT), )
 	obj-$(CONFIG_VIDEO_ATOMISP) += css2300_build/
-- 
1.7.9.5

